src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClient.java

```diff
@@ -278,9 +285,7 @@ public ChangeInfo get(EnumSet<ListChangesOption> options) throws RestApiExceptio
 
     @Override
     public ChangeInfo get() throws RestApiException {
-        EnumSet<ListChangesOption> options = EnumSet.allOf(ListChangesOption.class);
-        options.remove(ListChangesOption.CHECK);
```

Comment:
I do not remember why I remove CHECK here, but I think we should keep it here (and not in `ListChangesOptionByVersion`).

Reply:
See my comment above: `CHECK` is not a level fo detail of the change but is a *trigger* of a consistency check on the backend. I am not really sure why we implemented it in this way in Gerrit.

Reply:
Can I resolve this?